libnavigation-ui/build.gradle

```diff
@@ -115,6 +115,7 @@ dependencies {
   testImplementation dependenciesList.robolectric
   testImplementation dependenciesList.json
   testImplementation dependenciesList.androidxTestCore
+  testImplementation dependenciesList.androidxTestCore
```

Comment:
Duplicated ☝️ 

```suggestion
```

Reply:
This was supposed to be for ThemeSwitcher. But that migration requires either, java androidx.preferences, or to migrate ThemeSwitcher to kotlin.

Think it would be better to migrate ThemeSwitcher to kotlin first?